app/gui2/src/util/callTree.ts

```diff
@@ -388,12 +388,14 @@ export function getAccessOprSubject(app: Ast.Ast): Ast.Ast | undefined {
 
 export const ArgumentApplicationKey: unique symbol = Symbol('ArgumentApplicationKey')
 export const ArgumentInfoKey: unique symbol = Symbol('ArgumentInfoKey')
+export const ArgumentNameShownKey: unique symbol = Symbol('ArgumentNameShownKey')
 declare module '@/providers/widgetRegistry' {
   export interface WidgetInput {
     [ArgumentApplicationKey]?: ArgumentApplication
     [ArgumentInfoKey]?: {
       appKind: ApplicationKind
       info: SuggestionEntryArgument | undefined
+      [ArgumentNameShownKey]?: boolean
```

Comment:
We use symbols as keys in WidgetInput as it's meant to be extended by just anyone, including user's custom widgets. But there is no need to use them inside `ArgumentInfoKey`'s object

So we could replace it with normal field. Or, what I think is more elegant (while also more complex): Make `ArgumentNameShownKey` a key of WidgetInput (not `ArgumentInfoKey`'s value) and define it not here but near WidgetArgumentName - as it is this widget who defines the value, and callTree by itself does not care about argument being displayed or not.

But I'm ok with the first solution if the second will have some unforeseen difficulties.